nengo/builder/ensemble.py

```diff
@@ -189,6 +189,13 @@ def build_ensemble(model, ens):
         encoders = npext.array(ens.encoders, min_dims=2, dtype=rc.float_dtype)
     if ens.normalize_encoders:
         encoders /= npext.norm(encoders, axis=1, keepdims=True)
+    n_nans = np.sum(np.isnan(encoders))
+    if np.any(np.isnan(encoders)):
```

Comment:
The calculation of `n_nans` could be move into the if clause or the if condition could be based on `n_nans`.

Reply:
Good catch!